hw6 is done

Ltaudato1 opened the pull request.

Ltaudato1 committed:
hw6 is done

Ltaudato1 committed:
hw7 is done

Ltaudato1 committed:
hw8 is done

damir2407 commented on scrapper/src/main/java/edu/java/service/jpa/JpaLinkUpdateService.java:
```diff
@@ -0,0 +1,78 @@
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class JpaLinkUpdateService implements LinkUpdateService {
```
На будущее: благодаря spring data jpa ты можешь юзая JpaRepository сделать банальный круд в 3 строки, вместо ручного внедрения EntityManager'а и написания методов руками. Будет выглядить более ORM'но что ли)

Это если что на будущее. Советую все же почитать и исправить. В рамках этой дз - аппрув

Ltaudato1 committed:
hw9

Ltaudato1 committed:
hw10

Ltaudato1 committed:
updated workflow

Ltaudato1 committed:
added jars

Ltaudato1 committed:
ACTUALLY added jars

Ltaudato1 committed:
updated docker-files

Ltaudato1 committed:
temporary fix

Ltaudato1 committed:
temporary fix v2

почему-то именно на гите докер билд вообще не проходит, хотя локально всё работает вообще отлично

Ltaudato1 committed:
refactor hw7

Ltaudato1 committed:
Merge pull request #11 from Ltaudato1/hw10

hw10

Ltaudato1 committed:
Merge pull request #10 from Ltaudato1/hw9

hw9

Ltaudato1 committed:
Merge pull request #9 from Ltaudato1/hw8

hw8 is done

Ltaudato1 committed:
Merge pull request #8 from Ltaudato1/hw7

hw7 is done

Ltaudato1 committed:
solving merge conflict pt.1

Ltaudato1 committed:
Merge branch 'hw6' of https://github.com/Ltaudato1/tinkoff_edu_spring into hw6
